pipeline_steps/multi_node_aio_prepare.groovy

```diff
@@ -2,6 +2,8 @@ def prepare() {
   common.conditionalStage(
     stage_name: 'Prepare Multi-Node AIO',
     stage: {
+      common.prepareRpcGit(env.RPC_BRANCH)
```

Comment:
With this added, do we still need https://github.com/rcbops/rpc-gating/blob/e3167bc59f32e0af071678ccb32136b7342d36b2/pipeline_steps/multi_node_aio_prepare.groovy#L45?